src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Color color = colorSupplier.getRandomColor();
+    private int number = new Random().nextInt(100);

```

Comment:
they should be created not once but each time when method `getRandomBall` is being called, so move them to the method